everyvoice/wizard/basic.py

```diff
@@ -475,7 +487,7 @@ def effect(self) -> None:
             Panel(
                 f"You've finished configuring your dataset. Your files are located at {config_dir.absolute()}",
                 title="Congratulations 🎉",
-                subtitle="Next Steps Documentation: https://docs.everyvoice.ca/guides",
+                subtitle="Next Steps Documentation: https://docs.everyvoice.ca/stable/guides",
```

Comment:
ah - weird, I guess our re-routes aren't working properly. I thought we had a docs.everyvoice.ca/* redirect to stable. maybe worth looking into